ovs/src/main/java/org/dcsa/conformance/standards/ovs/checks/OvsChecks.java

```diff
@@ -13,234 +14,185 @@
-
-    checks.add(
-        JsonAttribute.customValidator(
-            "Validate universalServiceReference exists and "
-                + "matches in JSON response if request parameter has universalServiceReference",
-            body ->
-                validateParameter(
-                    body,
-                    sspSupplier,
-                    OvsFilterParameter.UNIVERSAL_SERVICE_REFERENCE,
-                    "*/universalServiceReference")));
-
-    checks.add(
-        JsonAttribute.customValidator(
-            "Validate vesselIMONumber exists and"
-                + " matches in JSON response if request parameter has vesselIMONumber",
-            body ->
-                validateParameter(
-                    body,
-                    sspSupplier,
-                    OvsFilterParameter.VESSEL_IMO_NUMBER,
-                    "*/vesselSchedules/*/vesselIMONumber")));
-
-    checks.add(
-        JsonAttribute.customValidator(
-            "Validate vesselName exists and"
-                + " matches in JSON response if request parameter has vesselName",
-            body ->
-                validateParameter(
-                    body,
-                    sspSupplier,
-                    OvsFilterParameter.VESSEL_NAME,
-                    "*/vesselSchedules/*/vesselName")));
-
-    checks.add(
-        JsonAttribute.customValidator(
-            "Validate carrierVoyageNumber exists and "
-                + "matches in JSON response if request parameter has carrierVoyageNumber",
-            body ->
-                validateParameter(
-                    body,
-                    sspSupplier,
-                    OvsFilterParameter.CARRIER_VOYAGE_NUMBER,
-                    "*/vesselSchedules/*/transportCalls/*/carrierExportVoyageNumber",
-                    "*/vesselSchedules/*/transportCalls/*/carrierImportVoyageNumber")));
-
     checks.add(
-        JsonAttribute.customValidator(
-            "Validate universalVoyageReference exists and"
-                + " matches in JSON response if request parameter has universalVoyageReference",
-            body ->
-                validateParameter(
-                    body,
-                    sspSupplier,
-                    OvsFilterParameter.UNIVERSAL_VOYAGE_REFERENCE,
-                    "*/vesselSchedules/*/transportCalls/*/universalImportVoyageReference",
-                    "*/vesselSchedules/*/transportCalls/*/universalExportVoyageReference")));
+      JsonAttribute.customValidator(
+        "Every response received during a conformance test must contain schedules",
+        body -> {
+          Set<String> validationErrors = new LinkedHashSet<>();
+          checkServiceSchedulesExist(body)
+            .forEach(
+              validationError ->
+                validationErrors.add(
+                  "CheckServiceSchedules failed: %s".formatted(validationError)));
+          if (validationErrors.isEmpty()) {
+            return Set.of();
+          }
```

Comment:
Line 41-43 can be removed, because line 44 does the same (return empty Set, or with entries).